internal/plan/plan_test.go

```diff
@@ -1017,3 +1064,65 @@ func (s *S) TestMarshalLayer(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(string(out), Equals, string(layerBytes))
 }
+
+func (s *S) TestGetCommand(c *C) {
+	cmdTests := []struct {
+		summary         string
+		command         string
+		cmdArgs         []string
+		expectedCommand []string
+		error           string
+	}{
+		{
```

Comment:
Let's please join these two lines so we get `}{{`, and move the slice out of the function, so we gain two indentation level for everything below and keep the function small. See the example of planTests above for reference.